### ISSUE TITLE
JET Data

### ISSUE THREAD

**Issue by reedacus25** [codefornola/nola-transit-map#20]
I think it would be helpful to have JET bus routes included with the RTA data to have a more comprehensive transit status, since there is overlap between JET and RTA in certain parts of the metro, despite the lack of fare transfers. The map for their live status is [here](https://jetapp.jptransit.org/map).

**Comment by naf419** [comment]
the static gtfs data (route geometries) are not available from jp transit directly, but they are available from ride (https://rideneworleans.org/opendata/gtfs/)

the actual bus location data is available via some light reverse engineering, but im pretty sure its not appropriate to contribute here... jp transit doesn't respond to official requests for the feed, so i dont know where that leaves us...

**Comment by reedacus25** [comment]
Appreciate the response @naf419!

Glad to learn about a new open data format with GTFS.

That said, I'm assuming that the feeds advertised for JeT [here](https://www.transit.land/operators/o-jefferson~parish#11.03/29.9832/-90.1825) would be functional, both [RT](https://jetapp.jptransit.org/gtfsrt/) and [static](https://www.transit.land/feeds/f-jefferson~parish), but I'm assuming (part of) the reluctance is that there is no license to access the data, as it appears there is for NORTA? As you said, the RT feeds look trivial, but the static feeds seem to be a magic link someone unearthed or published?

**Comment by naf419** [comment]
thanks for that link, dont know how i missed it. it does look a lot more legit than my reverse engineered version of their javascript app's "encrypted" api keys...

**Comment by bhelx** [comment]
this should be deployed thanks to #38 

do we have route info for these vehicles?

**Comment by reedacus25** [comment]
@bhelx comparing against [JET's canonical map](https://jetapp.jptransit.org/map), as well as in Le Pass, the data appears to be in lock step, there also happens to be exceptionally limited service today. Weekend service is extremely poor, coupled with [parade service interruptions](https://jptransit.org/wp-content/uploads/2025/02/JPT-Mardi-Gras-Rider-Alert-Schedule.pdf), explains the general sparsity this morning. [E5](https://jptransit.org/wp-content/uploads/2018/11/JPT-E5-System-Map-Schedule.pdf), [E8](https://jptransit.org/wp-content/uploads/2018/11/JPT-E8-System-Map-Schedule.pdf), [W8](https://jptransit.org/wp-content/uploads/2018/10/JPT-W8-System-Map-Schedule.pdf), [W10](https://jptransit.org/wp-content/uploads/2018/10/JPT-W10-System-Map-Schedule.pdf] as examples.

But given the JET data inclusion now, the system as a whole looks far more holistic for the entire metro, and riders transferring systems. So I greatly appreciate @naf419 's help on getting this across the finish line!

**Comment by naf419** [comment]
> do we have route info for these vehicles?

yes. they ignored my 10/29/2024 email requesting the static GTFS data for routes, but we grab a copy of it that is kindly hosted by [RIDE New Orleans](https://rideneworleans.org)
